DfE.FindInformationAcademiesTrusts/Pages/Index.cshtml

```diff
@@ -10,31 +10,29 @@
   <div class="app-banner app-banner--l">
     <div class="dfe-width-container">
       <div class="govuk-grid-row">
-        <div class="govuk-grid-column-one-half">
-          <form class="govuk-form-group app-search govuk-!-margin-bottom-0" asp-page="Search" method="post" role="search" aria-label="sitewide">
-            <h1 class="govuk-label-wrapper">
-              <label class="govuk-label govuk-label--xl app-banner--heading govuk-!-margin-bottom-8" for="@Model.PageSearchFormInputId">
-                Find information about academies and trusts
-              </label>
-            </h1>
-            <p class="govuk-body">Use this tool to find information on trusts and schools, including academies.</p>
-            <details class="govuk-details" data-module="govuk-details" data-testid="what-you-can-find-list">
-              <summary class="govuk-details__summary">
-                <span class="govuk-details__summary-text">
-                  <span>Establishments you can search</span>
-                </span>
-              </summary>
-              <ul class="govuk-list govuk-details__text govuk-list--bullet">
-                <li class="govuk-!-margin-left-5">academies</li>
-                <li class="govuk-!-margin-left-5">free schools</li>
-                <li class="govuk-!-margin-left-5">local authority maintained schools</li>
-                <li class="govuk-!-margin-left-5">special schools</li>
-                <li class="govuk-!-margin-left-5">trusts</li>
-              </ul>
-            </details>
-            <partial name="_PageSearchForm" model="Model"/>
-          </form>
-        </div>
+          <div class="govuk-grid-column-one-half">
+                    <h1 class="govuk-heading-xl app-banner--heading govuk-!-margin-bottom-8">
```

Comment:
DfE design system is altering the font size and line width here - we need to re-override it

https://design-system.service.gov.uk/styles/type-scale/#if-you-only-want-to-set-the-font-size